inc/IXR_Library.php

```diff
@@ -822,6 +826,8 @@ function getXml() {
  */
 class IXR_Date {
 
+    const XMLRPC_ISO8601 = 'Ymd\TH:i:s';
```

Comment:
Hmm. Its weird. The spec example gives it without any time zone info, but at the same time calls it ISO8601. According to https://en.wikipedia.org/wiki/ISO_8601#Time_zone_designators there are different ways to represent the timezone, but it seems not to mention that it is optional.

Since the original problem in #2883 was your Java parser failing, could you check if it would work with DateTime:ATOM format? I would prefer having the timezone info intact.